src/renderers/shared/ReactDOMFrameScheduling.js

```diff
@@ -23,10 +23,33 @@
 import type {Deadline} from 'ReactFiberReconciler';
 
 var invariant = require('fbjs/lib/invariant');
+var ExecutionEnvironment = require('fbjs/lib/ExecutionEnvironment');
 
 // TODO: There's no way to cancel these, because Fiber doesn't atm.
 let rAF: (callback: (time: number) => void) => number;
 let rIC: (callback: (deadline: Deadline) => void) => number;
+
+if (!ExecutionEnvironment.canUseDOM) {
+  global.requestAnimationFrame = function(
```

Comment:
Take a look at the code below. We export `rAF` and `rIC` from this file. So we should just assign those variables instead of globals.